src/AppMenuSearch.cc

```
@@ -0,0 +1,386 @@
+                searchMenu->hide();
+            }
+        });
+        searchMenu->addAction(newAction);
+        resultCount++;
+    }
+
+    Q_EMIT repositionRequested();
+    searchMenu->setUpdatesEnabled(true);
+}
+
+void AppMenuSearch::clear(QMenu *searchMenu)
```

Clang (14.0.6)

[warning] method 'clear' can be made static

(readability-convert-member-functions-to-static)